### ISSUE TITLE
508-defect-2 [SCREENREADER]: JAWS does not announce some sections of the multi-step forms when loaded

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#317]
## Issue
JAWS + IE11 does not announce some of the page changes when we load new sections of the 526 v2 all claims form. This can be problematic to non-sighted users or users who are on the diminished vision spectrum, because they rely on verbal cues to when we load new content.

This seems to be happening when there is no `FETCH` action undertaken on the previous screen. If I am using the `Back >` button to review the previous page, no ajax call is made, and nothing is announced. If I interact with the form in some meaningful way on a page and press `Continue`, JAWS announces the progress bar as expected.

Screenshots attached below illustrating the Network tab and logic flow.

## Related Issues
* https://github.com/department-of-veterans-affairs/vets.gov-team/issues/16782
* https://github.com/department-of-veterans-affairs/va.gov-team/issues/4679
* Please see the comment thread on that older, closed ticket. It has relevant information about what has been researched and eliminated as possible sources.

## Acceptance Criteria
* As a screen reader user, I want to hear the progress bar announced each time I advance or go back to the previous step in the form. I should hear the progress bar read aloud whether I have changed anything on the page or not.

## Environment
* Win10
* IE11
* JAWS

## WCAG or Vendor Guidance (optional)
* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

---
![screen shot 2019-02-06 at 12 04 27 pm](https://user-images.githubusercontent.com/934879/52363208-d3ea5c00-2a07-11e9-8d19-3885d81ffbf8.png)

---
![screen shot 2019-02-06 at 12 04 20 pm](https://user-images.githubusercontent.com/934879/52363220-da78d380-2a07-11e9-9935-5cf604bdc687.png)

**Comment by alexpappasoddball** [comment]
Removing this from Triage and assigning to Tools and Improvements as it is already in their epic.

**Comment by jenstrickland** [comment]
The first time I loaded the page it did one thing, and the second time it did another. Perhaps this is related to caching? I can't recall just now what the experience was. Noting this anyway, per conversation with @1Copenut. 

The unpredictable behavior is confusing. 

- If the page will screen read differently each time I visit it, I may not realize I'm on the same page I was previously.
- Also, if the pattern changes from one cached page to the next uncached, that inconsistency will interfere with understanding the content and where I am in the flow.

**Comment by 1Copenut** [comment]
This issue and #4679 seem like they're related, and symptomatic of the same issue. Chris V. took a look at the issue when I initially flagged it, but couldn't see anything definitive. It was suggested in a meeting today it could be a caching issue when we don't load data or have to repaint the page. I'll do some looking to see if I can get an idea of potential impact, given this is an IE11 + JAWS issue.

**Comment by rianfowler** [comment]
https://www.w3.org/TR/wai-aria-1.1/#progressbar
![Screen Shot 2020-06-29 at 14 38 15](https://user-images.githubusercontent.com/4998130/86043094-2f125980-ba16-11ea-95fa-47af51c756a5.png)

Look at any form like [this](https://staging.va.gov/education/submit-school-feedback/service-information)
- remove the progressbar role from the heading (<FormNav>)
- work with trevor to test on staging

**Comment by Clark-W-Griswold** [comment]
PR here: 
https://github.com/department-of-veterans-affairs/vets-website/pull/13314

**Comment by rianfowler** [comment]
@1Copenut this should be ready for testing. let us know if this doesn't work.

**Comment by 1Copenut** [comment]
@rianfowler && @Clark-W-Griswold I retested this one on the Education feedback form. Our phantom behavior with IE11 + JAWS is still happening. I was able to add a piece to the puzzle however.

When I commit the following steps, it seems focus is not being moved to the next heading:

1. Turn on JAWS in IE11
2. Navigate to the staging.va.gov/education/submit-school-feedback/application-information/ form
3. Identify you're filling out the form for yourself (Me)
4. TAB to Continue and press SPACE
5. Fill out the data on the 1 of 5: Application Information
6. TAB to Continue and press SPACE to move to the next application information screen
7. Now, TAB to the Back button and press SPACE
8. Now, TAB to the Continue button again, and press SPACE
9. Verify you hear "Continue right double angle bracket link" read aloud

You have have to repeat steps 6-9 several times, but it's possible to re-create. This is odd because we're using `<button>` for our back and continue steppers, so I'm not sure where this link is coming from.

**Comment by Clark-W-Griswold** [comment]
Associated PR:
https://github.com/department-of-veterans-affairs/vets-website/pull/13314

**Comment by meganhkelley** [comment]
@bkjohnson @cvalarida are y'all actively working on this? If not we can move to `Ready`

**Comment by bkjohnson** [comment]
We are not.

**Comment by 1Copenut** [comment]
It's possible we'll take care of this one with the work @bkjohnson is doing on the [iOS bug issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/12323).

**Comment by taylorkaren** [comment]
Comments above indicate this may be related to https://github.com/department-of-veterans-affairs/va.gov-team/issues/4679, which was closed on 3/8/2022, and https://github.com/department-of-veterans-affairs/va.gov-team/issues/12323, which was closed on 9/16/2020.

Retest this now to see if it is still an issue?

**Comment by caw310** [comment]
Closing as we are no longer supporting IE11
